mcs/class/Facades/netstandard/apicheck.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
```

Comment:
is this file intentional included in this PR?

Reply:
yes